earth_enterprise/src/common/khxml/khxml_unittest.cpp

```diff
@@ -13,6 +13,133 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
+#include <sys/stat.h>
+#include <cstdlib>
+#include <fstream>
+#include <memory>
+#include <sstream>
+#include <string>
+
+#include "khxml.h"
+#include "khdom.h"
+
+using namespace std;
+using namespace khxml;
+
+// These tests are focused on testing the Open GEE XMl code, not the Xerces
+// code. Thus, they do not try to hit every possible error condition; rather
+// they hit at least one error condition in every function and make sure Open
+// GEE handles it gracefully.
```

Comment:
Minor: It might help devs unfamiliar with the code if it mentions specifically which functions are being tested and where they're located. I ended up having to do a search to find them because I was looking in khxml.h.